README.md

```diff
@@ -20,6 +20,13 @@ was written correctly and much more.
 
 ![Etcher](https://raw.githubusercontent.com/resin-io/etcher/master/screenshot.png)
 
+Supported Operating Systems
+---------------------------
+
+- GNU/Linux
+- macOS 10.9 and later
```

Comment:
Curiosity question: What dependency do we have that only allows for macOS 10.9+? Electron itself is 10.8+, right?